### ISSUE TITLE
decrypt wd my book 

### ISSUE THREAD

**Issue by raminbabaie** [themaddoctor/linux-mybook-tools#35]
Hi,  , i am HDD repairing and Data recovery Technician  you can check me in linked plz send me massage i will send to you linked in address.
 
Hope you are well. I was reading your training about Decryption of data in WD  my passport HDD that a question came to my mind that we challenge with it everyday in our center .

 sometimes customers have password on their HDD and have forgotten it ,how we can access to the password  ?

the next question is , sometimes HDD has password and customers format HDD and the data become encrypted ,do you have any idea for finding the password?
we use data recovery tools .like mrt .wd marvel .i can find easily keyblock than module 25 or 38

by the way it was very useful thanks for your efforts and i hope i can also help people with your guidance .
i am looking forward to hearing from you.

**Comment by themaddoctor** [comment]
I can recover the disk key for the JMS538S and Symwave chips in 95% of cases, because I have special tools. For other chipsets, I can find passwords with a dictionary attack (but no guarantee).

For password recovery, I need some documentation to show that the client is the true owner of the disk.

If I understand your 2nd question, formatting does not change the disk key, unless the client does a special "quick erase" using the WD software. In that case, I can recover the key only for JMS538S chipsets.

**Comment by themaddoctor** [comment]
I hope that answers your questions.

**Comment by raminbabaie** [comment]
thanks for reply  .very good .can u help me for find password and decrypt data?. i dont use linux.i use data recovery software.

**Comment by raminbabaie** [comment]
my work is data recovery and i have very prb for this

**Comment by themaddoctor** [comment]
1. Show me the documentation that your client is the true owner of the disk.

2. Is it a MyBook or a Passport?

3. Which chipset is on the USB bridge card?

**Comment by raminbabaie** [comment]
hi. this is passport, and have symwave 6316 usb bridge ic ,and with wd
security software formated

**Comment by raminbabaie** [comment]
plz send me email ,i will send to you  documentation factor for client

**Comment by themaddoctor** [comment]
I'm sorry, but for the Symwave chip I cannot help if the drive was reencrypted with the WD software. It is not clear from your messages what has happened to the drive.

**Comment by raminbabaie** [comment]
No problem Thomas .I have another case symwave but not formatted .customer
just lost password.if you can help me .we in center very have problem for
this subject.

**Comment by raminbabaie** [comment]
Customers think that in the event of a forgotten password, they can format the hard disk and retrieve it with the usual software.

**Comment by raminbabaie** [comment]
[``](https://mega.nz/#!jihwjYQJ!cyBeXrCduq-T6ATq3-qy2CKlNTUN2z3jZkzlzM1kDRY)
this is keyblock for lost password hdd.thanks for help

**Comment by raminbabaie** [comment]
https://mega.nz/#!jihwjYQJ!cyBeXrCduq-T6ATq3-qy2CKlNTUN2z3jZkzlzM1kDRY

**Comment by themaddoctor** [comment]
Send proof of true ownership and I will send you the key to the encryption. Then you will have to follow the instructions in "Mounting encrypted WD disks in linux", using the section on Symwave in XTS mode.

**Comment by themaddoctor** [comment]
I don't think that you and I agree on what the word "format" means.

**Comment by raminbabaie** [comment]
Plz give me email address to send file .I can't load here

**Comment by raminbabaie** [comment]
Erase with wd security I mean .

**Comment by themaddoctor** [comment]
Erasing with WD security software DESTROYS the keyblock, and then the key is lost forever.

**Comment by themaddoctor** [comment]
this is a temporary email. I cannot guarantee that it works. joclutitre@wemel.site

**Comment by raminbabaie** [comment]
Hmmmmmm.ok I got it .where I can send file to u?

**Comment by themaddoctor** [comment]
joclutitre@wemel.site

**Comment by themaddoctor** [comment]
Sorry, that email cannot handle RAR files.

**Comment by raminbabaie** [comment]
i have sent it .plz check it

**Comment by raminbabaie** [comment]
ok now i send picture .plz check again

**Comment by themaddoctor** [comment]
It only handles text files. Encode the jpg in base64 and send that.

**Comment by raminbabaie** [comment]
i sent

**Comment by themaddoctor** [comment]
Where do you work?

**Comment by raminbabaie** [comment]
Emdadrayaneh ,laptop and hdd repair and recovery center in Shiraz ,Iran.
my name is Ramin

**Comment by themaddoctor** [comment]
What you sent is a receipt or work order from your own business. It does not show that the client owns the disk. Proof of ownership could be the original receipt or the registration for the warranty on the WD website.

**Comment by raminbabaie** [comment]
you are right, but the problem is that we live in a country that is under
strict sanctions and we dont have any access to international marketing
like buying from the WD website. this receipt is the only proof that we can
get from the customer in our center and all of the centers in Iran. our
purpose is just to help the customer to not lose the data.i hope you
can understand my situation. thank you.

**Comment by raminbabaie** [comment]
I'm waiting for your answer can you help me?

**Comment by themaddoctor** [comment]
My temporary email has expired. What is your email address?

**Comment by raminbabaie** [comment]
Araz.babaie@gmail.com

**Comment by raminbabaie** [comment]
Plz check this email .
blthgiwzjluimtyl@protonmail.com

**Comment by themaddoctor** [comment]
No, sorry. It is illegal in my country to share software that can be used to break into computer systems.

**Comment by raminbabaie** [comment]
ok .no prb .but can you at least tell me name of the software or which hash
i can use to extract key password. then i can research and learn myself

**Comment by raminbabaie** [comment]
thank you for help bro.:):)
